packages/react-dropdown/src/index.test.js

```diff
@@ -546,3 +546,63 @@ it('Children provided trough items should be present', () => {
   expect(children.prop('data-itemid')).toBe(131);
   expect(children.length).toBe(1);
 });
+
+it('should clear the previous selected items', () => {
+  const wrapper = mount(
+    <Dropdown items={[{ id: 10, label: 'Hello world' }]} initialIsOpen={true}>
+      {({ getInputProps, clearSelection }) => (
+        <div>
+          <Input {...getInputProps()} />
+          <button onClick={clearSelection}>Reset</button>
+        </div>
+      )}
+    </Dropdown>
+  );
+
+  wrapper
+    .find('ul li')
+    .at(0)
+    .simulate('click');
+
+  expect(
+    wrapper
+      .find('input')
+      .at(0)
+      .props().value
+  ).toBe(10);
+
+  wrapper.find('button').simulate('click');
+
+  expect(
+    wrapper
+      .find('input')
+      .at(0)
+      .props().value
+  ).toBe('');
+});
+
+it('onChange should be called with the newly selected items', () => {
+  const onChangeFn = jest.fn();
+
+  const wrapper = mount(
+    <Dropdown
+      items={items}
+      initialIsOpen={true}
+      selectedItems={[items[0]]}
+      onChange={onChangeFn}
+    >
+      {({ getInputProps }) => <Input {...getInputProps()} />}
+    </Dropdown>
+  );
+
+  wrapper
+    .find('ul li')
+    .hostNodes()
+    .at(1)
+    .simulate('click');
+
+  expect(onChangeFn).toHaveBeenCalledWith(
```

Comment:
We should also make sure the selection doesn't change in the UI when the component is used with `selectedItems` (to make sure `selectedItems` is always the source of truth)